### PR TITLE
added Expense removed with delete icon in history tab

### PR DESCRIPTION
**Pull request by siddharth0-0**
`Added employee name in my-report/team-report cards and minor css fixes`
image

my-re

`When expense is removed from report, the history section should show the title as `Expense removed` with delete icon in red color`
de
del

`Update `Exit Fyle App` popover to new design`
image
Screenshot_20220113-184457_Fyle Staging

### CODE REVIEW

File: src/app/app.component.ts [hunk 1]
```diff
@@ -69,26 +70,37 @@ export class AppComponent implements OnInit {
...
+      component: PopupAlertComponentComponent,
+      cssClass: 'pop-up-in-center',
     });
 
-    if (popupResults === 'primary') {
+    await popover.present();
+
+    const { data } = await popover.onWillDismiss();
+
+    console.log('da', data);
```
Comment (Dimple16) [inline src/app/app.component.ts:102]: remove

File: src/app/fyle/my-view-report/my-view-report.page.scss [hunk 2]
```diff
@@ -145,18 +149,33 @@
...
   }
 
-  &--date-info-block {
-    margin-top: 8px;
+  &--submitted-date {
+    font-weight: 500;
+    font-size: 12px;
+    &__text {
+      color: $light-grey;
+      opacity: 0.8;
```
Comment (Dimple16) [inline src/app/fyle/my-view-report/my-view-report.page.scss:157]: do we need this?

File: src/app/fyle/my-view-report/my-view-report.page.scss [hunk 3]
```diff
@@ -145,18 +149,33 @@
...
+    }
+
+    &__date {
+      color: $pure-white;
+    }
   }
 
-  &--submitted-date {
+  &--employee-name-block {
+    padding: 0 5px;
```
Comment (Dimple16) [inline src/app/fyle/my-view-report/my-view-report.page.scss:166]: 6?

**Review by Dimple16 — CHANGES_REQUESTED** [review]

File: src/app/fyle/my-view-report/my-view-report.page.scss (hunk 2 above)
Comment (siddharth0-0) [reply src/app/fyle/my-view-report/my-view-report.page.scss:157]: Yes opacity was added  to make text look similar to figma design

**Review by iravisankar — APPROVED** [review]

**Review by Dimple16 — APPROVED** [review]

**Pull request merged**